data_model.md

```diff
@@ -196,6 +196,9 @@ Each Forecast must be associated with a Site which contains important
 information such as location. A Site object must be created before a
 Forecast can be associated with it.
 
+#### Single-valued forecasts
```

Comment:
`Single-valued` means only GHI, or a forecast without uncertainty?  I think simply `Forecasts` is better here, with a note in the text that probabilistic forecasts require a few additional parameters.  Later, when contrasting a probabilistic forecast with a forecast without uncertainty, the technical term is `deterministic`.

Reply:
We started using "single-valued" instead of "deterministic" in response to stakeholder feedback. The idea was that a 50th percentile forecast is one of the most commonly issued single valued forecasts, and it's not right to call that a deterministic forecast.

Reply:
Using `single-valued` here is confusing, when I read it, because the term could mean a forecast with only a single value at each point in time, a forecast with only a single time point, or a forecast of a single quantity (e.g. GHI) as opposed to GHI and power.  Maybe this heading is better without the adjective.

I'm OK using `single-valued` rather than `deterministic` when we're contrasting probabilistic and non-probabilistic forecasts.